src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -390,6 +410,8 @@ public AzureAppConfigurationOptions ConfigureRefresh(Action<AzureAppConfiguratio
                 _changeWatchers.Add(item);
             }
 
+            RefreshOptions = refreshOptions;
```

Comment:
Can we expose only `RegisterAllEnabled` instead of `RefreshOptions` ?

Reply:
From what I understood, we need `RefreshInterval` in the provider like where I used it in `InitializeAsync` because of how the options class works. We don't know the final set of selected key values or what the final refresh interval is until we get into the provider. 

Is it still better to just do those 2 values instead of `RefreshOptions`?

Reply:
If we need both, then sure we can keep RefreshOptions.